app/views/hackdays/_project_list.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if hackday.votable %>
```

Comment:
sure. but i still think that will be less repeated lines of code then what you have here.